_data/sites.json

```diff
@@ -17547,6 +17547,18 @@
         ]
     },
 
+    {
+        "name": "Tapas",
+        "url": "https://tapas.io/profile/settings",
+        "difficulty": "easy",
+        "notes": "Scroll to the bottom, click \"Delete Account\", and click through the prompts.  You'll need to re-enter your password.",
+        "email": "feedback@tapas.io",
```

Comment:
```suggestion
```
We suggest not to add an email if it's not needed for the deletion, because it might confuse users.

Reply:
Makes sense!  Fixed in ec668ee